CMakeLists.txt

```diff
@@ -60,66 +58,36 @@ MESSAGE(${PYTHON_INCLUDE_DIRS})
 #  Boost library
 #
 
-MESSAGE("Looking for Boost libraries...")
+MESSAGE(STATUS "Looking for Boost libraries...")
 SET(Boost_NO_BOOST_CMAKE TRUE) # to fix the problem of finding python37 libs
 SET(Boost_USE_STATIC_LIBS=OFF)
 SET(Boost_USE_MULTITHREADED=OFF)
 SET(Boost_USE_STATIC_RUNTIME=OFF)
 
-# Adapt it according to your system
-#FIND_PACKAGE(Boost 1.73.0 REQUIRED)
-FIND_PACKAGE(Boost 1.82.0 REQUIRED)
-#FIND_PACKAGE(Boost REQUIRED)
+FIND_PACKAGE(Boost 1.74.0 REQUIRED COMPONENTS regex python${LIBRA_Python3_VERSION})

```

Comment:
again, the latest version of Boost I could find at the moment is 1.82.0 - why to roll back to 1.74.0 version?
If this is a user's system limitation, this should be noted (e.g. users could adapt their CMakeLists file)

Reply:
It is a minimal supported version. I have this version. If you have 1.82, it will be also found.